routes/api-routes.js

```diff
@@ -4,18 +4,35 @@ const config = require('../config/speechConfig.js');
 const multipartMiddleware = multipart({uploadDir: config.files.uploadDirectory});
 const db = require('../models');
 const { publish } = require('../controllers/publishController.js');
-const { getClaimList, resolveUri } = require('../helpers/lbryApi.js');
+const { getClaimList, resolveUri, getClaim } = require('../helpers/lbryApi.js');
 const { createPublishParams, validateApiPublishRequest, validatePublishSubmission, cleanseChannelName, checkClaimNameAvailability, checkChannelAvailability } = require('../helpers/publishHelpers.js');
 const errorHandlers = require('../helpers/errorHandlers.js');
-const { postToStats, sendGoogleAnalytics } = require('../controllers/statsController.js');
+const { postToStats } = require('../controllers/statsController.js');
 const { authenticateOrSkip } = require('../auth/authentication.js');
 
+function addGetResultsToFileData (fileInfo, getResult) {
```

Comment:
This is only used in one place? Possibly doesn't need to be a function? (Not sure, fine either way)

Reply:
yeah, only used in one place at the moment (and perhaps something that should happen lower in the model layer), but I am trying to compartmentalize code into functions where possible for ease of reading as well as testing.